includes/wf_crm_webform_postprocess.inc

```diff
@@ -1689,7 +1689,17 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
    * This happens during form validation and sets a form error if unsuccessful
    */
   private function submitLivePayment() {
-    $result = wf_civicrm_api('contribution', 'transact', $this->contributionParams());
+    $contributionParams = $this->contributionParams();
+
+    // Only if #installments = 1, do we process a single transaction/contribution. #installments = 0 in CiviCRM Core means open ended commitment;
+    $numInstallments = wf_crm_aval($contributionParams, 'installments', 1, $strict = TRUE);
+    if ($numInstallments === 1) {
```

Comment:
=== is actually working properly - but will change this to == (I just verified that == is working fine for three cases: 0, 1 and >1 (e.g. 10)